Gododir/gen.go

```diff
@@ -318,8 +328,8 @@ func generate(service *service) {
 
 	fmt.Println(commandArgs)
 
-	if _, err := runner(autorest); err != nil {
-		panic(fmt.Errorf("Autorest error: %s", err))
+	if _, stderr, err := runner(autorest); err != nil {
+		failed = append(failed, fmt.Sprintf("%s: autorest error: %s: %s", service.Fullname, err, stderr))
```

Comment:
As mentioned offline, this is a race condition 😄

Reply:
Fixed